vignettes/Using_rerddap.Rmd

```diff
@@ -0,0 +1,671 @@
+---
+title: "Using rerddap to Access Data from ERDDAP Servers"
+author: "Roy Mendelssohn and Scott Chamberlain"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{rerddapVignette}
+  %\VignetteEngine{knitr::rmarkdown}
+  \usepackage[utf8]{inputenc}
+---
+```{r initialize, echo = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+library(rerddap)
+```
+---
+
+## Introduction
+
+`rerddap` is a general purpose <span style="color:blue">R</span> client for working with <span style="color:blue">ERDDAP</span> servers.  <span style="color:blue">ERDDAP</span> is a web service developed by Bob Simons of NOAA.  At the time of this writing, there are over fifty <span style="color:blue">ERDDAP</span> servers  (though not all are public facing) providing access to literally petabytes of data and model output relevant to oceanography, meteorology, fisheries and marine mammals, among other areas. <span style="color:blue">ERDDAP</span> is a simple to use, RESTful web service, that allows data to be subsetted and returned in a variety of formats.
```

Comment:
I like the blue styling for `ERDDAAP`  text, but it does look like a link since the actual links are the same color. I wonder if you could use a different color, or bold or italicize?